spec/queries/claims/claims_query_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+describe Claims::ClaimsQuery do
+  subject(:claims_query) { described_class.call(params:) }
+
+  let(:params) { {} }
+
+  describe "#call" do
+    it "returns all visible claims, ordered by created at date" do
+      _internal_claim = create(:claim)
+      draft_claim = create(:claim, :draft)
+      submitted_claim = create(:claim, :submitted)
+
+      expect(claims_query).to match_array([draft_claim, submitted_claim])
```

Comment:
`match_array` doesn't consider order. To check order you need `eq`

Reply:
I forgot we care about the order here...